backend/infrahub/core/graph/__init__.py

```diff
@@ -1 +1 @@
-GRAPH_VERSION = 21
+GRAPH_VERSION = 22
```

Comment:
I saw some mention of this elsewhere where it needed to be bumped. Is this the final version or do we need 23 here now?